streaming/stream_transfer_task.cc

```diff
@@ -176,13 +176,6 @@ future<> send_mutations(lw_shared_ptr<send_info> si) {
 }
```

Comment:
Why is this reverted? the changelog should explain.

Reply:
Sorry, left over during debug. I will remove it.